cmd/manifest/init-v2.go

```diff
@@ -519,10 +517,20 @@ func inferDeploySection(cwd string) (*model.DeployInfo, error) {
 }
 
 func inferDevsSection(cwd string) (model.ManifestDevs, error) {
-	files, err := os.ReadDir(cwd)
+	dirEntries, err := os.ReadDir(cwd)
 	if err != nil {
 		return nil, err
 	}
+
+	files := make([]fs.FileInfo, 0, len(dirEntries))
```

Comment:
Why is this needed?

Reply:
due to a deepasource complain

Reply:
Can we extract it into a function then?

Reply:
reverted due to issue when pulling last updates